spm_results_export.m

```diff
@@ -28,14 +27,12 @@ function spm_results_export(SPM,xSPM,TabDat)
     if exist(fullfile(SPM.filepath,'html')) ~= 7 

```

Comment:
Hi @cmaumet, this is the comment I mentioned before I think is made redundant by the base64 branch - would you like me to change it still?